src/mesh/mesh.cpp

```diff
@@ -475,11 +475,6 @@ Mesh::Mesh(ParameterInput *pin, ApplicationInput *app_in, Packages_t &packages,
   }
 
   ResetLoadBalanceVariables();
-
-  // Output variables in use in this run
-  if (Globals::my_rank == 0) {
-    std::cout << "#Variables in use:\n" << *(resolved_packages) << std::endl;
-  }
```

Comment:
Isn't this included in another PR? Did that not get merged?

Reply:
No, that one's still in queue.  I pinged some people, probably it will still go in before this one and I can avoid having to correct my mistake here.